fetch-utils.js

```diff
@@ -1,12 +1,39 @@
-const SUPABASE_URL = '';
-const SUPABASE_KEY = '';
+const SUPABASE_KEY = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJyb2xlIjoiYW5vbiIsImlhdCI6MTY0MTk0ODIwNCwiZXhwIjoxOTU3NTI0MjA0fQ.jqvK0NeN6qBna_UCyT_2jgahLwFZ0_I-7T3QMQHJhvo';
+
+const SUPABASE_URL = 'https://nvkeilaiioresqjruzbf.supabase.co';
 
 const client = supabase.createClient(SUPABASE_URL, SUPABASE_KEY);
 
 export async function getUser() {
     return client.auth.session();
 }
 
+export async function getWorkshops() {
+    const response = await client
+        .from('workshops')
+        .select('*, participants (*)');
+    
+    return checkError(response);    
+}
+
+export async function deleteParticipant(id) {
+    const response = await client 
+        .from('participants')
+        .delete()
+        .match({ id: id })
+        .single();
+
+    return checkError(response);
+}
+
+export async function createPartiipant(participant) {
+    const response = await client
+        .from('participants ')
```

Comment:
there is an extra space between participants and the last single quote here.  I don't know if it is throwing anything off but your insert call is not working at the moment so worth checking out.